Src/WitsmlExplorer.Frontend/components/Modals/ReportModal.tsx

```diff
@@ -207,9 +179,18 @@ export const useGetReportOnJobFinished = (jobId: string): BaseReport => {
                 )
               );
             } else {
-              setReport(jobInfo.report);
-              if (jobInfo.report.downloadImmediately === true) {
-                downloadData(jobInfo.report.reportItems, jobInfo.report.title);
+              const report = await JobService.getReport(jobId);
```

Comment:
This shouldn't be fetched here in the else. Instead, replace the jobInfo fetch as we no longer need to fetch the jobInfo to get the report.

Reply:
fixed